### PR TITLE
Luigi side nav

### PR DESCRIPTION
**Pull request by dariadomagala**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ..

**todos**:
- [x] show selected node
- [x] Make it TS
- [x] set max width for texts in left nav
- [x] change the icon from sap to kyma 
- [x] add an icon for settings panel
- [x] footer
- [x] external nodes (in progress)
  - [x] add an icon on the right to show that it's external
- [x] fix csses to not show margin if navigation is not present
- [x] add an option to make navigation compact
- [x] ^ show children for compact navigation
- [x] add namespace chooser 
  - [x] get namespaces on opening the dropdown
- [x] navigation is broken in few places (extensions, helmreleases etc.)
- [x] show full navigation node's name on hover (PW)
- [x] show 3 dots at the end if the navigation node's name is too long (PW)

**Related issue(s)**
See #1939 
https://github.com/SAP/fundamental-react/pull/1457

### CODE REVIEW

**Comment by dariadomagala** [comment]
Few items are not selected, but are also not crucial, will be handled in the follow up, since the PR is pretty big already

**Comment by Wawrzyn321** [comment]
First navigation node is hidden due to `position: fixed`.
image

**Review by Lyczeq — COMMENTED** [review]
Why namespace details are specified here  `core-ui/src/resources/Namespaces/index.js` and here `core-ui/src/resources/Namespaces/namespaceOverviewNode.ts`?

Edit:
Oh nvm

**Review by Wawrzyn321 — APPROVED** [review]

File: core-ui/src/components/App/useInitTheme.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { useRecoilValue } from 'recoil';
+import { themeState } from 'state/preferences/themeAtom';
+
+// we have to initially load the themeState to
```
Comment (IwonaLanger) [inline core-ui/src/components/App/useInitTheme.ts:4]:
```suggestion
// you must initially load the themeState to
```

File: core-ui/src/components/App/useInitTheme.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+//let know the App what theme to choose
```
Comment (IwonaLanger) [inline core-ui/src/components/App/useInitTheme.ts:5]:
```suggestion
//let the App know what theme to choose
```

**Review by IwonaLanger — APPROVED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Review by IwonaLanger — DISMISSED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**